code/modules/cards/cardhand.dm

```diff
@@ -0,0 +1,149 @@
+/obj/item/toy/cards/cardhand
+	name = "hand of cards"
+	desc = "A number of cards not in a deck, customarily held in ones hand."
+	icon = 'icons/obj/playing_cards.dmi'
+	icon_state = "none"
+	w_class = WEIGHT_CLASS_TINY
+	worn_icon_state = "card"
+
+/obj/item/toy/cards/cardhand/Initialize(mapload, list/cards_to_combine)
+	. = ..()
+
+	if(!LAZYLEN(cards_to_combine) && (mapload && !LAZYLEN(cards)))
+		CRASH("[src] is being made into a cardhand without a list of cards to combine")
+
+	if(mapload && LAZYLEN(cards)) // these cards have not been initialized
+		for(var/card_name in cards)
+			var/obj/item/toy/singlecard/new_card = new (loc, card_name)
+			new_card.update_appearance()
+			cards_to_combine += new_card
+		cards = list() // reset our cards to an empty list
+
+	if(LAZYLEN(cards_to_combine)) // these cards are already initialized
+		for(var/obj/item/toy/singlecard/new_card in cards_to_combine)
+			new_card.forceMove(src)
+			cards += new_card
+
+	register_context()
+	update_appearance()
+
+/obj/item/toy/cards/cardhand/suicide_act(mob/living/carbon/user)
+	user.visible_message(span_suicide("[user] is slitting [user.p_their()] wrists with \the [src]! It looks like [user.p_they()] [user.p_have()] a crummy hand!"))
+	playsound(src, 'sound/items/cardshuffle.ogg', 50, TRUE)
+	return BRUTELOSS
+
+/obj/item/toy/cards/cardhand/examine(mob/user)
+	. = ..()
+	for(var/obj/item/toy/singlecard/card in cards)
+		if(HAS_TRAIT(user, TRAIT_XRAY_VISION))
+			. += span_notice("You scan the cardhand with your x-ray vision and there is a: [card.cardname]")
+		var/marked_color = card.getMarkedColor(user)
+		if(marked_color)
+			. += span_notice("There is a [marked_color] mark on the corner of a card in the cardhand!")
+
+/obj/item/toy/cards/cardhand/add_context(atom/source, list/context, obj/item/held_item, mob/living/user)
+	if(istype(held_item, /obj/item/toy/cards/deck))
+		var/obj/item/toy/cards/deck/dealer_deck = held_item
+		if(dealer_deck.wielded)
+			context[SCREENTIP_CONTEXT_LMB] = "Deal card"
+			context[SCREENTIP_CONTEXT_RMB] = "Deal card faceup"
+			return CONTEXTUAL_SCREENTIP_SET
+		context[SCREENTIP_CONTEXT_LMB] = "Recycle cards"
+		return CONTEXTUAL_SCREENTIP_SET
+
+	if(istype(held_item, /obj/item/toy/singlecard))
+		context[SCREENTIP_CONTEXT_LMB] = "Combine cards"
+		context[SCREENTIP_CONTEXT_RMB] = "Combine cards faceup"
+		return CONTEXTUAL_SCREENTIP_SET
+
+	return NONE
+
+/** TODO let someone select a card from the cardhand using the radial menu
+/obj/item/toy/cards/cardhand/on_offered(mob/living/carbon/offerer)
+/obj/item/toy/cards/cardhand/on_offer_taken(mob/living/carbon/offerer, mob/living/carbon/taker)
```

Comment:
commented out code

Reply:
This was deliberate since I wanted to use these for a later feature where you can give people cards from a cardhand.  If this is a problem I can delete it, I just wanted to keep a reference to the procs needed instead of having to hunt them down again.

Reply:
Removed with 321616b6816fbdadbc9cab7b77547446949272c0